src/engine/input/key_mapping/InputActionItem.cs

```diff
@@ -90,6 +90,10 @@ public override void _Ready()
         inputEventsContainer = GetNode<HBoxContainer>(InputEventsContainerPath);
         addInputEvent = GetNode<Button>(AddInputEventPath);
 
+        var tooltipCallbacks = new List<ToolTipCallbackData>();
```

Comment:
why?
I don't need the tooltipCallbacks anyways, but I have to provide the method with something.

Reply:
It is needed to store the mouse enter and exit callbacks for the tooltips (display and hide). I'm afraid that it will get freed by garbage collection here (I think) and the tooltips stop responding to mouse events on the Control.

Reply:
I've looked through the code and it seems that the callbackDatas can be safely disposed by the GC. It is only used to help the caller to make changes to the element afterwards, which I don't need.

Reply:
Wait, I've found a path that needs this object. Changing...